websites/O/OpenFING/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Since this presence uses `iframe.ts`, please also provide `iFrameRegExp`. This should be a regular expression that matches the domain(s) the iFrame is located at.